content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,357 @@ schema: olm.channel
 
 Here, a channel is generated for each template channel which differs by minor version, and each channel has a `replaces` edge from the predecessor channel to the next-lesser minor bundle version. Please note that at no time do we transgress across major-version boundaries with the channels, to be consistent with [the semver convention](https://semver.org/) for major versions, where the purpose is to make incompatible API changes.
 
+## Composite Template
+You can use a `composite template` to generate a valid FBC for multiple catalogs at a time. A `composite template` is an input schema that accepts a configuration schema. Catalog maintainers can use a `composite template` to build multiple versions of a catalog while maintaining fewer files. Operator authors can use a `composite template` to contribute their operator to multiple catalogs.
+
+### Usage
+```sh
+opm alpha render-template composite [flags] <filename>
+```
+
+| Flag                | Description                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -h, --help          | prints help/usage information                                                          |
+| -o, --output <type> | the output format, can be `yaml` or `json`                                             |
+| -c, --composite-config <file> | File to use as the composite configuration file (default "catalog/config.yaml") |
+| --skip-tls-verify   | skip TLS certificate verification for container image registries while pulling bundles |
+| --use-http          | use plain HTTP for container image registries while pulling bundles                    |
+| --validate          | whether or not the created FBC should be validated (i.e 'opm validate') (default true) |
+| --container-tool <tool> | container tool to be used when rendering veneers (should be an equivalent replacement to docker - similar to podman) (default "docker") |
+
+## Specification
+The composite veneer consists of the following schemas:
+- `olm.composite`
+    - This schema defines what, where, and how to build a particular contribution
+- `olm.composite.catalogs`
+    - This schema defines a catalog and the constraints for building the catalog
+
+The `olm.composite.catalogs` cue schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+The `olm.composite` cue schema is:
+```
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
```

Comment:
We probably should include s/Component/Contribution/g in the "schema trim" jira. We weren't always consistent as we evolved this thing. ☹️